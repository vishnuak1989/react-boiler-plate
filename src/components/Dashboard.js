import React from 'react'

const Dashboard=()=>(

  <div className="container">
    <h1> Dashboard </h1>
    
  </div>
);
export default Dashboard;
