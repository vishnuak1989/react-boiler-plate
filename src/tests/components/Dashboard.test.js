import React from 'react'
import  Dashboard  from '../../components/Dashboard'
import { shallow } from 'enzyme'


test("Should render Dashboard Page",()=>{
  const wrapper = shallow(<Dashboard />);
  expect(wrapper).toMatchSnapshot();
})
