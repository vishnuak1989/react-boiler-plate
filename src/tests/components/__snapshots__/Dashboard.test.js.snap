// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Dashboard Page 1`] = `
<div
  className="container"
>
  <h1>
     Dashboard 
  </h1>
</div>
`;
